### PR TITLE
Javascript Pizzeria, Sofias Week 1 individual assignment

### PR DESCRIPTION
**Pull request by sofia32057**
## Netlify link
[My Netlify link](https://sofias-pizza.netlify.app/)

### CODE REVIEW

File: .gitignore
Comment (AntonellaMorittu) [inline .gitignore:1]: .DS_Store is a file that stores custom attributes of its containing folder, such as folder view options, icon positions, and other visual information, and we don't need to share it in our project. Please add .DS_Store to your .gitignore file (if you don't have one, you need to create it) 👍

File: code/switch.js [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+
+// Prompt customer to confirm order
+const orderConfirmation = prompt(
+  `Ok, so you've ordered ${portionSize.toLowerCase()} portion of ${foodChoice} ${subType} for ${price} SEK. Sounds good?
+
+  - YES
+  - NO`
+);
+
+console.log("Order confirmation: ", orderConfirmation); // Log to console
```
Comment (AntonellaMorittu) [inline code/switch.js:306]: Please remove the the console.log() around the code. Generally they need to be deteled when the code has been tested and works as meant to 🧹

File: code/switch.js [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+// Step 5 - Order confirmation
+// Your code goes here
+
+// Prompt customer to confirm order
+const orderConfirmation = prompt(
+  `Ok, so you've ordered ${portionSize.toLowerCase()} portion of ${foodChoice} ${subType} for ${price} SEK. Sounds good?
+
+  - YES
+  - NO`
+);
```
Comment (AntonellaMorittu) [inline code/switch.js:304]: this step is a bit redundant, could as well skip to the last confirmation message 😄

**Review by AntonellaMorittu — APPROVED** [review]
Hi Sofia, you did a great job and I have very little to add apart from the comments I left in your code, overall your code meets the requirements and stretch goals, yay! ⭐ 
Please take a look at my tips :) Thank you ⭐

File: .gitignore
Comment (sofia32057) [reply .gitignore:1]: Thought I did this in one of the last commits on Feb 8. Doesn't it work?

**Pull request closed without merging**